### PR TITLE
GAWB-395: Choose type when copying from other workspace

### PR DESCRIPTION
**Pull request by MattPutnam**
- [x] **Submitter**: Rebase to develop. DO NOT SQUASH
- [x] **Submitter**: Make sure Swagger is updated if API changes
- [x] **Submitter**: Make sure documentation for code is complete
- [x] **Submitter**: Review code comments; remove done TODOs, create stories for remaining TODOs
- [x] **Submitter**: Include the JIRA issue number in the PR description
- [x] **Submitter**: Add description or comments on the PR explaining the hows/whys (if not obvious)
- [x] **Submitter**: Tell tech lead that the PR exists if s/he wants to look at it
- [x] **Submitter**: Anoint a lead reviewer (LR). **Assign PR to LR**
- [x] **LR**: Initial review by LR and others.
- [x] Comment / review / update cycle:
  - Rest of team may comments on PR at will
  - **LR assigns to submitter** for feedback fixes
  - Submitter updates documentation as needed
  - Submitter rebases to develop again if necessary
  - Submitter makes further commits. DO NOT SQUASH. **Reassign to LR** for further feedback
- [x] **Tech lead**: sign off
- [x] **LR**: sign off
- [x] **Assign to submitter** to finalize
- [x] **Submitter**: Squash commits, rebase if necessary
- [x] **Submitter**: Verify all tests go green, including CI tests
- [x] **Submitter**: Merge to develop 
- [x] **Submitter**: Delete branch after merge
- [x] **Submitter**: Check configuration files in Jenkins in case they need changes
- [x] **Submitter**: **Test this change works on dev environment after deployment**. YOU own getting it fixed if dev isn't 
  working for ANY reason!
- [x] **Submitter**: Verify swagger UI on dev server still works after deployment
- [x] **Submitter**: Inform other teams of any API changes via hipchat and/or email
- [x] **Submitter**: Mark JIRA issue as resolved once this checklist is completed

### CODE REVIEW

**Comment by davidangb** [comment]
I think the delete-entity dialog also needs updating.

**Comment by MattPutnam** [comment]
In what way?

**Comment by davidangb** [comment]
it no longer shows all entity types, and has no selector for choosing the type to delete. If your intention was that the delete-entity dialog only shows the type in the current table, that's fine ... but that's a behavior change and needs to be OK-ed.

File: src/cljs/org/broadinstitute/firecloud_ui/common/components.cljs [hunk 1]
```diff
@@ -275,9 +275,12 @@
...
             (when (seq causes)
-              [:div {}
-               [:div {} (str "Cause" (when (> (count causes) 1) "s") ":")]
-               (map (fn [cause] [CauseViewer cause]) causes)])
+              (let [num-hidden (- (count causes) 4)]
+                [:div {}
+                 [:div {} (str "Cause" (when (> (count causes) 1) "s") ":")]
+                 (map (fn [cause] [CauseViewer cause]) (take 4 causes))
+                 (when (pos? num-hidden)
+                   [:div {} (str num-hidden " not shown")])]))
```
Comment (davidangb) [inline src/cljs/org/broadinstitute/firecloud_ui/common/components.cljs:285]: What's happening here? Does the ErrorViewer now show a max of 4 causes? I don't necessarily object to this, but it feels ... unrelated. Can you talk about why this is necessary?

Comment (MattPutnam) [inline src/cljs/org/broadinstitute/firecloud_ui/common/components.cljs:285]: If you try to import a sample set, but already have some of the samples, you'll get a separate error for each conflict.  So I capped it.

**Comment by MattPutnam** [comment]
It was my intention to only see the current type in the delete dialog.  This may help with the "out of order" problem.

**Comment by davidangb** [comment]
code looks good, works good for me locally. :+1: for the code. I think the UI treatment of selecting types for copy is good.

This PR identified three behavior changes:
- selecting type for copy
- limiting delete dialog to a single type
- capping errors displayed

all of these need to get OK'ed by PO!

**Pull request merged**